src/site/content/en/handbook/quick-start/index.md

```diff
@@ -1,6 +1,6 @@
 ---
 layout: handbook
-title: Quick start guide
+title: Content quick start guide
```

Comment:
Or something that tells me what the quick start is for, like "Contributor quick start guide" or "Quick start guide for adding content". If I just landed on this page (without seeing the content handbook landing page), I wouldn't know what this doc is about.